client/reader/following-edit/subscribe-form-result.jsx

```diff
@@ -1,44 +1,43 @@
-//const debug = require( 'debug' )( 'calypso:reader:following:edit' );
-
-// External dependencies
-const React = require( 'react' ),
-	classNames = require( 'classnames' );
-
-// Internal dependencies
-const ListItem = require( 'reader/list-item' ),
-	Icon = require( 'reader/list-item/icon' ),
-	Title = require( 'reader/list-item/title' ),
-	Description = require( 'reader/list-item/description' ),
-	Actions = require( 'reader/list-item/actions' ),
-	FollowButton = require( 'components/follow-button/button' ),
-	SiteIcon = require( 'components/site-icon' );
-
-var FollowingEditSubscribeFormResult = React.createClass( {
-
-	propTypes: {
-		url: React.PropTypes.string.isRequired,
-		isValid: React.PropTypes.bool.isRequired,
-		onFollowToggle: React.PropTypes.func.isRequired
-	},
-
-	render: function() {
-		const message = ! this.props.isValid
-			? this.translate( 'Not a valid URL' )
-			: this.translate( 'Follow this site' );
-		const classes = classNames( 'is-search-result', { 'is-valid': this.props.isValid } );
-
-		return (
-			<ListItem className={ classes }>
-				<Icon><SiteIcon size={ 48 } /></Icon>
-				<Title>{ this.props.url }</Title>
-				<Description>{ message }</Description>
-				<Actions>
-					<FollowButton following={ false } onFollowToggle={ this.props.onFollowToggle } />
-				</Actions>
-			</ListItem>
-			);
-	}
-
-} );
-
-module.exports = FollowingEditSubscribeFormResult;
+/**
+* External dependencies
+*/
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+* Internal dependencies
+*/
+import { translate } from 'i18n-calypso';
```

Comment:
probably want to use the `localize` HOC here instead of calling translate directly. That way when the user lang changes, the component will re-render. Not a giant deal, but how we tend to do it.